python/hsfs/feature_descriptive_statistics.py

```diff
@@ -0,0 +1,201 @@
+#
+#   Copyright 2023 Hopsworks AB
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import json
+import humps
+from typing import Optional, Mapping
+from hsfs.util import FeatureStoreEncoder
+
+
+class FeatureDescriptiveStatistics:
```

Comment:
Should this be a public class for the user? It is mostly internal no?

Reply:
What is public or not, it's a bit confusing to me.
With _public_ do you mean that it is expected that users may instantiate a class by themselves?
In that case, it is internal yes. Users don't need to create instances manually. However, they can see and interact with the instances of this class that we create and return anyways. 

I moved it to core.

Reply:
Ja I rather meant that unlike FG or FV it is mostly something that we use as wrappers of dictionary to make our life simpler, rather than a fundamental concept of the feature store. Thanks